Source/DataSources/GeoJsonDataSource.js

```diff
@@ -189,7 +189,7 @@ define([
 
             var description = properties.description;
             if (!defined(description)) {
-                entity.description = new CallbackProperty(createDescriptionCallback(properties, nameProperty), true);
+                entity.description = new CallbackProperty(createDescriptionCallback(describe, properties, nameProperty), true);
```

Comment:
Would it be better to have the provided `describe` return a `Property` instance and be called instead of creating a `CallbackProperty` every time?  It's almost the same as you have now, but I think it would be more flexible.  For example, the way this is currently set up, there's no good way (as far as I can tell) to supply an asynchronous describe function, but if you could return a custom property, that would be easy.  What do you think?